### PR TITLE
Script to update examples 

### PR DESCRIPTION
**Pull request by zregvart**
Updates the `example/data/trusted_tekton_tasks.yml` and `acceptance/samples/policy-input-golden-container.json` files. Added the updated files and aligned the golang version in `.tool-versions` file to the `go.mod`.

### CODE REVIEW

File: hack/refresh-examples.sh [hunk 1]
```diff
@@ -0,0 +1,89 @@
+#!/bin/env bash
+# Copyright 2022 Red Hat, Inc.
```
Comment (lcarva) [inline hack/refresh-examples.sh:2]: Can we just use `Copyright The Enterprise Contract Contributors` like we do in ec-cli?

**Review by lcarva — APPROVED** [review]
This is great! Can you also update the docs here: https://github.com/enterprise-contract/ec-policies/blob/main/acceptance/samples/README.md

File: hack/refresh-examples.sh (hunk 1 above)
Comment (zregvart) [reply hack/refresh-examples.sh:2]: Tots, not sure where I got this from...

Comment (zregvart) [reply hack/refresh-examples.sh:2]: Actually all scripts in `hacks` are doing Red Hat, Inc... I'll do that in a separate PR...

**Review by lcarva — APPROVED** [review]

**Pull request merged**